metals/src/main/scala/scala/meta/internal/bsp/BspConnector.scala

```diff
@@ -76,16 +71,6 @@ class BspConnector(
             .map(Some(_))
         case ResolvedBspOne(details) =>
           bspServers.newServer(workspace, details).map(Some(_))
-        case ResolvedMultiple(_, availableServers) =>
```

Comment:
This will now never happen. As I explained, if there is no longer an explicit choice resolved. we default to Bloop, which means it will only ever resolve one when doing the initial connection.